UpdateDevices/UpdateDevices.cs

```diff
@@ -332,7 +336,7 @@ private async Task<List<Microsoft.Graph.Models.ManagedDevice>> GetNewDeviceManag
         }
 
         
-        private static async Task<FunctionSettings> GetFunctionSettings()
+        private async Task<FunctionSettings> GetFunctionSettings()
         {
             MethodBase method = System.Reflection.MethodBase.GetCurrentMethod();
```

Comment:
Would like to get this logging to capture the method name to use in the application insights logs.